### PR TITLE
[keras/layers/activation] Standardise docstring usage of "Default to"

### PR DESCRIPTION
**Pull request by SamuelMarks**
This is one of many PRs. Discussion + request to split into multiple PRs @ #17748

### CODE REVIEW

File: keras/layers/activation/relu.py [hunk 1]
```diff
@@ -65,11 +65,11 @@ class ReLU(Layer):
       Same shape as the input.
 
     Args:
-      max_value: Float >= 0. Maximum activation value. Default to None, which
-        means unlimited.
-      negative_slope: Float >= 0. Negative slope coefficient. Default to 0.
-      threshold: Float >= 0. Threshold value for thresholded activation. Default
-        to 0.
+      max_value: Float >= 0. Maximum activation value. None means unlimited.
```
Comment (qlzh727) [inline keras/layers/activation/relu.py:68]: Please quote the None here.

**Review by qlzh727 — CHANGES_REQUESTED** [review]

File: keras/layers/activation/relu.py (hunk 1 above)
Comment (SamuelMarks) [reply keras/layers/activation/relu.py]: Fixed

**Comment by qlzh727** [comment]
Please resolve the merge conflicit.

**Comment by SamuelMarks** [comment]
@qlzh727 Resolved

**Review by qlzh727 — APPROVED** [review]

**Pull request merged**